appserver/ejb/ejb-container/src/main/java/com/sun/ejb/containers/EjbContainerUtilImpl.java

```diff
@@ -441,10 +422,6 @@ public void setActiveTxCache(Transaction jtx, Object cache) {
         txData.activeTxCache = cache;
     }
 
-    @Override
-    public Agent getCallFlowAgent() {
-        return callFlowAgent;
-    }
```

Comment:
So CallFlowAgent removal is not related to logging changes?  Then it should not be part of this PR.

Reply:
It is related, because it was used to fill some additional attributes in logs, but it doesn't work in last releases. It just confuses people. There's yet one related method where I just added some comment in my stash, because that is much more interesting and could be used also for other things - but that is certainly for another issue.
This one is dead, it was bound to abandoned architecture and thread management.